trestle/core/catalog/catalog_api.py

```diff
@@ -108,7 +108,7 @@ def read_additional_content_from_md(self,
             self._context.required_sections,
             label_map,
             self._context.sections_dict,
-            self._context.to_markdown
+            self._context.to_markdown,
         )
 
```

Comment:
@butler54 I see that there is a requirement of extra comma after last element in dictionary, array and in some functional calls (not all).